freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/handler/SetPasswordHandler.java

```diff
@@ -39,7 +38,7 @@ public void accept(Event<SetPasswordRequest> setPasswordRequestEvent) {
             SetPasswordResult result = new SetPasswordResult(request);
             request.getResult().onNext(result);
         } catch (Exception e) {
-            request.getResult().onError(new OperationException(e));
+            request.getResult().onError(e);
```

Comment:
Can we add log statement also?

Reply:
we shouldn't log because we are passing the exception out. we should let the calling code figure out whether to log or not log. Otherwise, we'll get duplicate logs for the same exception and logs for exceptions that we are handling.